config/default.yaml

```diff
@@ -63,3 +63,4 @@ extensions:
 
 cache:
   ttl: 1m
+  cleanup: 1m
```

Comment:
Can you change this value to "cleanupInterval"? Or remove it.

Reply:
Done!